rootfs/api/models.py

```diff
@@ -612,10 +614,12 @@ def clone(self, release):
 
     @close_db_connections
     def create(self):
+
```

Comment:
let's remove these extra whitespace additions to this PR. Classes should have an extra newline, but not functions. Let's keep this focused :)